packages/fleet/lib/instances.ts

```diff
@@ -0,0 +1 @@
+export type FleetId = 'nango_runners';
```

Comment:
It is not mandatory. It is just to give an explicit `id` to things in general and, in this case, not assuming fleet is a singleton for ever. I know it is right now but for the cost of 3 lines of code it could save a lot of pain later if we ever need to split the fleet for some reason